docs/guides/understanding_the_mining_process.rst

```diff
@@ -356,7 +356,7 @@ zero value transfer transaction.
   >>> chain = klass.from_genesis(AtomicDB(), GENESIS_PARAMS)
   >>> vm = chain.get_vm()
 
-  >>> nonce = vm.state.get_nonce(SENDER)
+  >>> nonce = vm.state.account_db.get_nonce(SENDER)
```

Comment:
@tczpl  Everything in this example is actual code that is executed on every CI run. If you just copy this out and replace `>>> ` and `... ` with an empty string, you can put it in a file and run it and it should just work.

Reply:
Thank you. I found it in state.py in the github. But when I execute it as it is in the guide, it throws `AttributeError: 'ByzantiumState' object has no attribute 'get_nonce'`.  So I check the state.py in my local package in python that it does not exist. It seems that I have installed an old version of py-evm. So I close this issue~